backends/open-meteo.com.go

```diff
@@ -110,6 +118,8 @@ var (
 func (opmeteo *openmeteoConfig) Setup() {
 	flag.StringVar(&opmeteo.apiKey, "openmeteo-api-key", "", "openmeteo backend: the api `KEY` to use if commercial usage")
 	flag.BoolVar(&opmeteo.debug, "openmeteo-debug", false, "openmeteo backend: print raw requests and responses")
+	opmeteo.cache = make(map[string]cachedData) // initializing a cache
+	opmeteo.cacheDuration = 1 * time.Hour       // setting the cache duation to 1 hour
```

Comment:
Should the cache duration be configurable?

Reply:
Good point! Making the cache duration configurable would give users more flexibility. I can add that feature to allow them to set it according to their needs.